components/PoweredBy/index.css

```diff
@@ -0,0 +1,9 @@
+.as-powered-by--block {
```

Comment:
So we're starting the BEM naming? Great!

But if so, I don't think `block` is very explicit.